server/entity/item_behaviour.go

```diff
@@ -65,6 +67,22 @@ func (i *ItemBehaviour) Item() item.Stack {
 // Tick moves the entity, checks if it should be picked up by a nearby collector
 // or if it should merge with nearby item entities.
 func (i *ItemBehaviour) Tick(e *Ent) *Movement {
+	w := e.World()
+	pos := cube.PosFromVec3(e.Position())
+	blockPos := pos.Side(cube.FaceDown)
+
+	bl, ok := w.Block(blockPos).(block.Hopper)
+	if ok && !bl.Powered && bl.CollectCooldown <= 0 {
+		_, err := bl.Inventory(w, pos).AddItem(i.i)
+		if err != nil {
+			// We couldn't add any of the item to the inventory, so we ignore it.
+			return i.passive.Tick(e)
+		}
+
+		_ = e.Close()
+		bl.CollectCooldown = 4
```

Comment:
```suggestion
		bl.CollectCooldown = 8
```

4 redstone ticks => 8 ticks